news/2021-06-14-mca-ayim-2019.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Mapper's Choice Awards and A Year In Mapping 2019"
+date: 2021-06-14 00:00:00 +0000
+---
+
+Map connoisseurs rejoice, because the *Mappers' Choice Awards* AND *A Year In Mapping* (AYIM) are back and again and rearing to catch up with the scene!
+
+It's been a long time coming, but these two incredibly popular events are making a grand return in 2021, first focusing on catching up on 2019 during this June and July before moving on to cover all of 2020's maps later on in the year.
+
+With all-new redesigned websites for these two titanic contests, it's never been easier to get involved and have your say on what maps really made the game tick, so make sure to read on below for all the details, dates and everything else you need to become a part of things.
+
+
+## Schedule for MCA and AYIM 2019
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Nomination stage | 2021-06-14 00:00 to 2021-06-21 00:00 |
+| Voting stage | 2021-06-23 00:00 to 2021-07-07 00:00 |
+| Results Stream | 2021-07-10 / 2021-07-11 |
+| Results posted to Website | 2021-07-12 00:00 |
+
+## A Year In Mapping 2019
+
+AYIM is back at long last for another round of highlighting the achievements and statistics of the mapping community in 2019. As in previous editions, each mode will have a section of statistics such as who ranked the most sets, how many difficulties were ranked overall, the extremes in difficulty levels we saw ranked, and many, many more records than you can think of.
```

Comment:
maybe explain a little bit here about what AYIM is for people who don't know? link some past contest results briefly, explain briefly how the voting works, so on, currently this doesn't tell anyone all that much of import

remember these contests have been dead for 2 years, that's an eternity for a lot of people who play